unit_testing/unit_test.py

```diff
@@ -562,6 +562,27 @@
     print(str(sec) + chr(subsec) + " OK - CRPS CDF plot saved")
```

Comment:
I notice you have two items in 6b. This looks odd. Perhaps at line 562 and 564
```print(str(sec) + chr(subsec) + " OK - CRPS CDF plot saved")```
-->
```print("   " + " OK - CRPS CDF plot saved")```

Reply:
Separated into two units

Reply:
just need to update the 6b to 6c in the text.